napari/components/viewer_model.py

```diff
@@ -6,7 +6,7 @@
 from .dims import Dims
 from .layerlist import LayerList
 from .. import layers
-from ..util import colormaps
+from ..util import colormaps  # noqa F401
```

Comment:
do you get a flake8 warning if you omit the `# noqa` here?  (that would be concerning since it is indeed used!)

Reply:
Not sure what was it about. It's gone, in any case.